### DISCUSSION TITLE
Difference between 3.3.2 and 3.3.2bis?

### DISCUSSION THREAD

**Discussion by fblais** [SebastJava/mint-themes#41] [General]
Is the theme zipfile the same?
Only the installer script and readme got changed?

Merci!

**Comment by SebastJava** [comment]
_Salut François,_

Exactly. The theme is the same, only the INSTALLER.sh and README.md got changed. That release v3.3.2 got deleted and replaced with v3.3.2-bis.

As usual, I tried to think about any possible issue, and do some basic tests **before** publishing. This installer was just ultra-simple and working as expected. So, I happily published it. And then I checked it again, to see how it goes when someone tries to install it from GitHub. A real-life experience, in other words. And then I found out I can't just upload the installer directly on the release page. The installer had lost its execute permission... So, there I had made an automatic installer, but to find out I would have to tell the user to do `chmod +x INSTALLER.sh`. That's not what I call perfect simplicity.

So, I made the installation process just a bit different. I hope it is okay this time...

P.S.: I forgot to mention this theme is not ready for the new LinuxMint 21...

**Comment by SebastJava** [comment]
Well, the zipfile itself DID get changed, but only for these INSTALLER.sh and README.md
